tests/test__collection_api.py

```diff
@@ -3153,21 +3153,6 @@ def test__aggregate_graph_lookup_depth_filed_not_string(self):
             "Argument 'depthField' to $graphlookup must be a string",
             str(err.exception))
 
-    def test__aggregate_graph_lookup_dot_in_connect_from_field(self):
```

Comment:
Please add a test for dotted field in `as` if there isn't one (it should trigger a NotImplementedError)